### PR TITLE
PIM-5941: Fix tab navigation in import/export profiles

### PR DESCRIPTION
**Pull request by damien-carcel**
**Description**

When editing a product export profile, if a user saves then edits again the profile, he/she will be on the same tab than previously. However, this behavior doesn't work in "show" mode, the user being always on the "General properties" tab.

The goal of this PR is for the user to always be in the tab he/she decides, during show/edit transitions.

Note that this behavior will work for all exports, and also for imports but only if upload is not enable. If upload is enable, then only 

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | N/A
| Added Behats                      | :white_check_mark:
| Changelog updated                 | N/A
| Review and 2 GTM                  | :white_check_mark:
| Micro Demo to the PO (Story only) | :white_check_mark:
| Migration script                  | N/A
| Tech Doc                          | N/A

### CODE REVIEW

File: features/export/product-export-builder/export_products_with_attributes.feature [hunk 1]
```diff
@@ -79,3 +79,19 @@ Feature: Export products with only selected attributes
...
+    When I am on the "csv_footwear_product_export" export job edit page
+    And I visit the "Content" tab
+    And I filter by "sku" with operator "IN" and value "BOOT-1"
+    And I press the "Save" button
+    And I should not see the text "There are unsaved changes"
+    Then I should be on the "Content" tab
+    When I visit the "History" tab
+    And I press the "Edit" button
+    Then I should be on the "csv_footwear_product_export" export job edit page
+    And I wait 1 seconds
```
Comment (grena) [inline features/export/product-export-builder/export_products_with_attributes.feature]: no wait please

Comment (damien-carcel) [inline features/export/product-export-builder/export_products_with_attributes.feature]: There is already a spin. Functional problem is, before this code fix, if you are on a tab then edit the profile,you first are on the expected tab before UI finish loading, then only you are on the wrong tab.
The behavior is very short, but enough to allow the test to succeed when it should not. I did not find another solution than a wait, and didn't want to add `$this->wait()` in the corresponding context as it is not needed everywhere.

Comment (juliensnz) [inline features/export/product-export-builder/export_products_with_attributes.feature]: No wait. You need to find a proper solution

File: src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/_edit_headers.html.twig [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{% set title =  (entityName ~ '.edit')|trans ~ ' - ' ~ jobInstance.label ~ ' [' ~ jobInstance.code ~ ']' %}
+
+{% set left %}
+    <ul class="inline">
+        <li>{{ 'Job'|trans }}: {{ job.name|trans }}</li>
```
Comment (juliensnz) [inline src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/_edit_headers.html.twig]: please use a proper translation key

File: src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/_edit_headers.html.twig [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <li>{{ 'Connector'|trans }}: {{ jobInstance.connector|trans }}</li>
```
Comment (juliensnz) [inline src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/_edit_headers.html.twig]: same

File: src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/_edit_headers.html.twig (hunk 2 above)
Comment (damien-carcel) [inline src/Pim/Bundle/ImportExportBundle/Resources/views/JobProfile/_edit_headers.html.twig]: These are the translation keys that were already here. I just moved code.

File: features/export/product-export-builder/export_products_with_attributes.feature (hunk 1 above)
Comment (damien-carcel) [inline features/export/product-export-builder/export_products_with_attributes.feature]: I know that, but I didn't find another solution, for the reason I explain in my previous comment. Any suggestion would be much appreciated…

**Comment by juliensnz** [comment]
GTM

**Comment by juliensnz** [comment]
GTM

**Comment by solivier** [comment]
GTM

**Pull request merged**